lib/constable_web/controllers/auth_controller.ex

```diff
@@ -10,7 +10,8 @@ defmodule ConstableWeb.AuthController do
     UserInterest
   }
 
-  @permitted_email_domain Application.fetch_env!(:constable, :permitted_email_domain)
+  def permitted_email_domain, do: Application.fetch_env!(:constable, :permitted_email_domain)
```

Comment:
What do you think about making these functions private? That way, we don't expose them outside of this module, which keeps the behavior in-line with how the module attribute was working.